File: 3 - Estrutura de Decisão/Exercicio.24.py
#24. Faça um algoritmo para ler três números e ordene-os em ordem crescente.

num1 = float(input("Digite o primeiro número: "))
num2 = float(input("Digite o segundo número: "))
num3 = float(input("Digite o terceiro número: "))

menor = min(num1, num2, num3)
maior = max(num1, num2, num3)
meio = (num1 + num2 + num3) - menor - maior

print("Os números em ordem crescente são:", menor, meio, maior)

File: Livro - Módulo Java/3 - Estrutura de Decisão/Exercicio.24.java
// Faça um algoritmo para ler três números e ordene-os em ordem crescente

import java.util.Scanner;

public class Exercicio24 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o primeiro número: ");
        double n1 = scanner.nextDouble();

        System.out.print("Digite o segundo número: ");
        double n2 = scanner.nextDouble();

        System.out.print("Digite o terceiro número: ");
        double n3 = scanner.nextDouble();

        double menor = Math.min(Math.min(n1, n2), n3);
        double maior = Math.max(Math.max(n1, n2), n3);
        double meio = (n1 + n2 + n3) - menor - maior;

        System.out.println("Os números em ordem crescente são " + menor + " , " + meio + " e " + maior);

        scanner.close();
    }
}